wcfsetup/install/files/ts/WoltLabSuite/Core/Bbcode/Code.ts

```diff
@@ -0,0 +1,124 @@
+/**
+ * Highlights code in the Code bbcode.
+ *
+ * @author	Tim Duesterhus
+ * @copyright	2001-2019 WoltLab GmbH
+ * @license	GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @module	WoltLabSuite/Core/Bbcode/Code
+ */
+
+import * as Language from "../Language";
+import * as Clipboard from "../Clipboard";
+import * as UiNotification from "../Ui/Notification";
+import Prism from "../Prism";
+import * as PrismHelper from "../Prism/Helper";
+import PrismMeta from "../prism-meta";
+
+const CHUNK_SIZE = 50;
```

Comment:
```suggestion
const chunkSize = 50;
```
Constants should look like regular variables.

Reply:
I opted to make this a static class member and adjusted the casing.